test/xexp.jl

```diff
@@ -0,0 +1,39 @@
+@testset "xexp" begin
+    @test isnan(xexp(NaN))
+    @test isinf(xexp(Inf))
+    @test xexp(-Inf) == 0
+    @test xexp(0.0) == 1
+
+    @test isinf(xexp(710.0))
+    @test xexp(-746.0) == 0
+
+    x = linspace(708.4, 709.7, 25)
+    @test_approx_eq xexp.(x) exp.(x)
```

Comment:
No need to ping Stefan it's right here:
http://docs.julialang.org/en/latest/stdlib/test/#other-test-macros